.github/workflows/release.yml

```diff
@@ -51,9 +51,11 @@ jobs:
         uses: gradle/wrapper-validation-action@v1
       - name: sha256
         run: |
-          make buildplugin
           tag=$(git describe --tags --abbrev=0 --exact-match)
           release_version="${tag:1}"
+          echo "release_version=${release_version}" >> "$GITHUB_ENV"
```

Comment:
`release_version` is used in the next step so we need to set it in the environment.

Reply:
ye, was wondering about that, thanks for the explanation!